Adding Customer Churn Prediction

Rutikab12 opened the pull request:
This classification problem is solved using logistic regression.

Rutikab12 committed:
Adding Customer Churn Prediction

This classification problem is solved using logistic regression.

ashishsahu1 commented:
@Rutikab12 follow this folder structure #42 , also try to export a pkl file for your model

Rutikab12 commented:
> @Rutikab12 follow this folder structure #42 , also try to export a pkl file for your model

Okay.

Rutikab12 committed:
Added Customer Churn Prediction

This project will classify the customers of a bank who are likely to churn as per analysis.
